docs/guides/integration/alternative-indexes.md

```diff
@@ -106,21 +106,21 @@ publish-url = "https://pkgs.dev.azure.com/<ORGANIZATION>/<PROJECT>/_packaging/<F
 Then, configure credentials (if not using keyring):
 
 ```console
```

Comment:
If you're dropping the `$` then this should be `sh` (or `bash` like below, although none of these rely on any bashisms) not `console`.